tests/OSPSuite.FuncParser.Tests/OSPSuite.FuncParser.Tests.csproj

```diff
@@ -1,28 +1,28 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="Current">
   <PropertyGroup>
     <TargetFramework>netcoreapp2.2</TargetFramework>
-
     <IsPackable>false</IsPackable>
-
     <RunPostBuildEvent>Always</RunPostBuildEvent>
-
     <RootNamespace>OSPSuite.FuncParser.Tests</RootNamespace>
+    <FileUpgradeFlags>40</FileUpgradeFlags>
+    <UpgradeBackupLocation>C:\Users\domin\OneDrive\Dokumente\Projekte\OSPSuite.FuncParser\Backup\tests\OSPSuite.FuncParser.Tests\</UpgradeBackupLocation>
```

Comment:
what is this "UpgradeBackupLocation" with an absolute path?

Reply:
I am sorry but it looks like i did not notice that VS2019 modified the file. That is garbage content that should not be there. I will clean the file and commit the cleaned version.